client/extensions/woocommerce/app/settings/email/mailchimp/getting-started.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import { localize } from 'i18n-calypso';
+
+class MailChimpGettingStarted extends React.Component {
+
+	render() {
+		const { translate, onClick } = this.props;
+		const infoText = translate( 'Allow your customers subscribe to your MailChimp email list' );
+
+		const illustration =
+			<img
+				src={ '/calypso/images/illustrations/illustration-layout.svg' }
+				width={ 220 }
+				className="mailchimp__getting-started-illustration"
+			/>;
+
+		const connect = translate( 'Connect with your customers through MailChimp' );
+		const allow = translate( 'Allow customers to subscribe to your Email list' );
+		const send = translate( 'Send abandon cart emails' );
+		const create = translate( 'Create purchase-based segments for targeted campaigns' );
+		const gettingStarted = translate( 'Get started with MailChimp' );
+		const list = [ allow, send, create ];
+
+		return (
+			<div>
+				<Card className="mailchimp__getting-started-title">
```

Comment:
Guessing all of these classnames are in a different PR?

Reply:
Yes - they are in the master PR.